Artifacts/windows-vscode/Artifactfile.json

```diff
@@ -2,15 +2,28 @@
     "$schema": "https://raw.githubusercontent.com/Azure/azure-devtestlab/master/schemas/2016-11-28/dtlArtifacts.json",
     "title": "Visual Studio Code",
     "publisher": "rchaganti",
-    "description": "Installs Visual Studio Code",
+    "description": "Installs the latest stable version of Visual Studio Code",
     "tags": [
-        "Windows",
-        "VSCode",
-        "Visual Studio"
+      "Windows",
+      "VSCode",
+      "Visual Studio"
     ],
     "iconUri": "https://github.com/Azure/azure-devtestlab/raw/master/Artifacts/windows-vscode/vscode.png",
     "targetOsType": "Windows",
+    "parameters": {
+      "Version": {
+        "type": "string",
+        "displayName": "Version",
```

Comment:
Maybe Architecture would be a better name.  I think Version would be confused for X.X.X.X strings.

Reply:
Yes, it was considered. It just didn't look right, but we can definitely change it.

Reply:
I'd guess that all the folks using this will know about architecture.